sources/packages/web/src/types/StoreState.ts

```diff
@@ -0,0 +1,13 @@
+import {
+  InstitutionUserAndAuthDetails,
+  InstitutionLocationsDetails,
+} from "@/types";
+
+export interface RootState {
+  version: string;
+}
+
+export interface InstitutionLocationState {
+  myInstitutionAndUserDetailsState: InstitutionUserAndAuthDetails;
+  myInstitutionLocationsState: InstitutionLocationsDetails;
```

Comment:
Are the InstitutionLocationsDetails interface being reused here because we need all the data or just to avoid create a specific type.
IMO, we should add to state what is needed. If we don't need the entire object, please consider creating a specific object to hold the state.